runtime/cmake/caches/osx_x86-64.cmake

```diff
@@ -0,0 +1,37 @@
+################################################################################
+# Copyright (c) 2018, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+
+#TODO Platform hacks
+
+set(OMR_OPT_CUDA OFF CACHE BOOL "")
+
+
+set(OMR_GC_COMPRESSED_POINTERS OFF CACHE BOOL "")
+set(OMR_INTERP_COMPRESSED_OBJECT_HEADER OFF CACHE BOOL "")
+set(OMR_INTERP_SMALL_MONITOR_SLOT OFF CACHE BOOL "")
+set(J9VM_GC_CLASSES_ON_HEAP OFF CACHE BOOL "")
+set(J9VM_GC_COMPRESSED_POINTERS OFF CACHE BOOL "")
+set(J9VM_INTERP_COMPRESSED_OBJECT_HEADER OFF CACHE BOOL "")
+set(J9VM_INTERP_SMALL_MONITOR_SLOT OFF CACHE BOOL "")
+
+#TODO: this should be based off of the linux_x86-64 (non compressed refs) once it exists
+include(${CMAKE_CURRENT_LIST_DIR}/linux_x86-64_cmprssptrs.cmake)
```

Comment:
This also includes commom.cmake which sets the above caches ON. Will this override the above?

Reply:
If there is an existing cache entry (either created by a prior `set()`, or a `-D` command line option) then `set( ... CACHE)` does nothing (unless the `FORCE` option is used). So since we are setting these before the include, everything should be ok